JPS_Ontology/ontology/ontocompany/OntoCompany.owl

```diff
@@ -91,185 +99,274 @@
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/Factory"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/EnergyConsumptionPerUnitFloorArea"/>
+        <rdfs:comment>A relationship between a factory and the amount of energy used per unit floor area.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Energy Consumption Per Unit Floor Area</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasFloorArea -->
+
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasFloorArea">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/IndustrialFacility"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/FloorArea"/>
+        <rdfs:comment>A relation between an industrial facility and its gross floor area.</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>floor area</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasGeneratedHeat -->
+
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasGeneratedHeat">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/IndustrialFacility"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/GeneratedHeat"/>
         <rdfs:comment>A relationship between an entity and its annual heat emission.</rdfs:comment>
-        <rdfs:isDefinedBy>http://www.theworldavatar.com/kg/ontocompany</rdfs:isDefinedBy>
-        <rdfs:label>has generated heat</rdfs:label>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Generated Heat</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasGeneratedHeatPerPrinter -->
+
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasGeneratedHeatPerPrinter">
+        <rdfs:domain rdf:resource="http://www.theworldavatar.com/kg/ontocompany/PrintingPlant"/>
+        <rdfs:range rdf:resource="http://www.theworldavatar.com/kg/ontocompany/GeneratedHeatPerPrinter"/>
+        <rdfs:comment>A relationship between a printing plant and the average amount of heat emitted by each printer within the facility</rdfs:comment>
+        <rdfs:isDefinedBy rdf:resource="http://www.theworldavatar.com/kg/ontocompany/"/>
+        <rdfs:label>Generated Heat Per Printer</rdfs:label>
+    </owl:ObjectProperty>
+    
+
+
+    <!-- http://www.theworldavatar.com/kg/ontocompany/hasIndustrialFacilityProperty -->
+
+
+    <owl:ObjectProperty rdf:about="http://www.theworldavatar.com/kg/ontocompany/hasIndustrialFacilityProperty">
```

Comment:
I am not sure what this concept is supposed to represent - are we talking about properties as in "attributes" or property as in a land plot? Could you either rename this or add more details to the definition of this concept in <rdfs:comment>

Reply:
I have included additional explanation for this predicate.